src/App.js

```diff
@@ -1,28 +1,13 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
+
+// const calculator = new Calculator();
 
 function App() {
   return (
```

Comment:
- [ ] Kindly make sure all of the components are implemented as class components, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).) requirement.

Reply:
hi @ShadyShawkat , Thank you so much for the suggestion I'll definitely apply it I have some questions can we please arrange a little zoom meeting? please Thankyou :)

Reply:
Hey @caasperr, 
Sure, you can reach me on Slack.